tests/tests_automl/test_joblib_version.py

```diff
@@ -0,0 +1,61 @@
+import unittest
+import joblib
+import numpy as np
+import json
+import shutil
+import os
+
+from supervised import AutoML
+
+
+class TestJoblibVersion(unittest.TestCase):
+
+    automl_dir = "automl_testing"
+
+    def tearDown(self):
+        shutil.rmtree(self.automl_dir, ignore_errors=True)
+
+    def test_joblib_version(self):
```

Comment:
Please make two tests:
1. Test that checks if joblib version is saved in the framework file.
2. Test that injects incorrect version in the JSON file and raise exception on AutoML model load.